lib/lightning/credentials.ex

```diff
@@ -175,7 +178,7 @@ defmodule Lightning.Credentials do
 
     Multi.new()
     |> Multi.update(:credential, changeset)
-    |> derive_events(changeset)
+    |> manage_projects_association(credential, changeset)
```

Comment:
`create_credential` still calls `derive_events`, what is different about `update_credential`? Ideally they both call the same event generating function no?

Reply:
Reverted back to derive_events. Thanks